lib/oli/delivery/sections.ex

```diff
@@ -2090,9 +2087,14 @@ defmodule Oli.Delivery.Sections do
         inserted_at: now,
         updated_at: now,
         collab_space_config: revision.collab_space_config,
-        max_attempts: if is_nil(revision.max_attempts) do 0 else revision.max_attempts end,
+        max_attempts:
+          if is_nil(revision.max_attempts) do
+            0
+          else
+            revision.max_attempts
+          end,
```

Comment:
Maybe here we can use the short way of writing the if statement, something like --> `max_attempts: (if is_nil(revision.max_attempts), do: 0, else: revision.max_attempts)`. I'm not sure if it's necessary to enclose the if statement in "()".